src/components/todo-input/TodoInput.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <form class="input-model">
```

Comment:
This form component shouldn't be necessary. 
This component just renders an input.